crates/rolldown/tests/rolldown/function/export_mode/umd/complex_name_named/_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "config": {
+    "format": "umd",
+    "name": "test.module",
+    "exports": "named",
+    "banner": "import assert from \"node:assert\"",
+    "footer": "assert(globalThis.test.module.a === 1);"
```

Comment:
How about using `_test.mjs`? This seems to abuse the banner and footer.

```js
// __test.mjs
import assert from "node:assert"
import "./dist/main.js"
assert(globalThis.test.module.a === 1);
```

Reply:
Good idea,Fixed it.